presto-docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -60,6 +61,14 @@ Rename column ``id`` to ``user_id`` in the ``users`` table if table ``users`` an
 
     ALTER TABLE IF EXISTS users RENAME column IF EXISTS id to user_id;
 
+Change owner of table ``people`` to user ``alice``::
+
+    ALTER TABLE people SET AUTHORIZATION alice
+
+Allow everyone to drop and alter table ``people``::
```

Comment:
`Allow everyone with role public to drop and alter table ``people``::`

Not everyone might have role public. I guess it is a case in Hive, but it might not be somewhere else.